frontend/src/data/ipeds_318_45/query/values.spec.ts

```diff
@@ -17,7 +17,14 @@ const all_ethnicities = new Query(
   'WhiteAlone',
   'NonUSResident'
 )
-
+const all_edu = new Query(
```

Comment:
Why do we use direct strings here instead of the values from `queries/dimensions`?